option optimize_for = LITE_RUNTIME;

message ComposeGallantRequest
{
 required uint32 id = 1;
 required uint32 num = 2;
}
